docs/index.md

```diff
@@ -1,3 +1,41 @@
-# Open Podcast API specification hub
+# Open Podcast API
 
-This site contains the specifications and documentation for the Open Podcast API.
+Welcome! The Open Podcast API is an initiative aiming to provide a feature-complete synchronisation API specification for podcast (web) apps and user-focussed servers.
+
+Our goal indirectly serving end-users: providing an easy way to sync subscriptions, listening progress, favourites, queu(s) and more between different apps and online services, and the ability to switch providers without hassle. Our goal for developers: providing reliable, open specifications which are decentralisation-ready and are easy to implement. To reach these goals we're deveveloping these specs in a collaborative way and sharing them in an [OpenAPI](https://spec.openapis.org/oas/latest) format.
+
+## Supported feature set & 'alpha' status
```

Comment:
I don't think the vocabulary around alpha/beta is something that helps here. Its used in software and we are writing a spec here, not a software. We should name it: Its incomplete and its a draft, not a finished specification. We might change it at any time until we call it stable and a recommendation for implementation.

I am not fully aware how the w3c vocabulary is here, it might be worth checking.

Reply:
Fair comments about alpha/beta used for software not specs. But if AntennaPod were to implement this in production (we offer multiple options and in order to test in practice we should include OPA), how should it be labelled? We can't write something like 'Synchronise via Open Podcast API (draft)' - hence the idea to call it 'alpha' (analogous to the status of implementation).

Draft I would say is not applicable now at this point; the specs were draft when we were working on them. It signals that you're open for theory-based comments/feedback. Now that we release them and ask people to implement we've past that point.

The [W3C terminology](https://www.w3.org/2021/Process-20211102/#maturity-levels) that I could find (e.g. 'Candidate Recommendation') doesn't really feel relevant here either. (At least in my perspective we're not 'recommending' anyone to use/implement this - which is different from web standards.)

I don't like 'unstable' because it sounds negative (doesn't indicate we're building towards something) and we don't expect it to be unstable (we would've thought harder about different use-cases that might cause unstable behaviour).

If not alpha, I would then go for 'incomplete' or maybe even 'in-progress'. It's a temporary thing anyway - once we have a good set of 'core' endpoints/calls we should move to versioning.

Reply:
I think we need to keep the standard specification and the implementation apart, which is exactly why I don't want to mix the wordings here.

The specification can be work in progress, a draft or maybe a final recommendation for implementation. The software you build following this spec can be alpha (generally means not feature complete), beta (feature complete but not battle-tested) or stable. This does only makes sense if you compare the software to some kind of spec.

Following your example in antenna-pod, its fair to claim you are implementing the Open Podcast API, and if its not considered features complete or unstable, you can call this a alpha/beta feature independently from the state of the specification (This is btw happening quite often, for example http signatures are built into Mastodon/other ActivityPub implementations long before it got remotely "ready").

I agree to your concerns regarding the W3C terminology, so maybe going with "in-progress" is the best pick for now.

@JonOfUs Is this something you can agree with?

Reply:
'in-progress' sounds good to me, a bit more positive than 'incomplete' maybe.
It's fine for me to name it this way until the core is rather complete.